tests/phpunit/tests/db.php

```diff
@@ -1526,6 +1528,10 @@ public function test_prepare_with_placeholders_and_individual_args( $sql, $value
 		// phpcs:ignore WordPress.DB.PreparedSQL
 		$sql = $wpdb->prepare( $sql, ...$values );
 		$this->assertSame( $expected, $sql );
+
+		if ( is_string( $incorrect_usage ) && array_key_exists( 'wpdb::prepare', $this->caught_doing_it_wrong ) ) {
+			$this->assertStringContainsString( $incorrect_usage, $this->caught_doing_it_wrong['wpdb::prepare'] );
```

Comment:
This test method previously had just one assertion.

Per [Writing PHPUnit Tests - Using Assertions](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions):
> All PHPUnit assertions, as well as all WordPress custom assertions, allow for a `$message` parameter to be passed. This message will be displayed when the assertion fails and can help immensely when debugging a test. This parameter should always be used if more than one assertion is used in a test method.

Suggestion: Update lines 1530 and 1533 to include a third `(string)` parameter which, if the assertion should fail, explains what went wrong.

Reply:
Good point, has been added.